src/features/common/Layout/UserLayout/UserLayout.tsx

```diff
@@ -52,22 +59,50 @@ function LanguageSwitcher() {
       }
     }
   }, []);
+  //to  get new pin
+  const handleNewPin = async () => {
+    try {
+      const response = await putAuthenticatedRequest<SupportPin>(
+        '/app/profile/supportPin',
+        undefined,
+        token
+      );
+      if (response) {
+        setSupportPin(response?.supportPin);
```

Comment:
When this is updated, you should maintain a sync with the user object in UserPropsContext.